### PR TITLE
Add TanzuNet upload to pipeline

### PR DESCRIPTION
**Pull request by bmdoil**

### CODE REVIEW

File: ci/concourse/vars/greenplum-database-release.prod.yml [hunk 1]
```diff
@@ -16,3 +16,10 @@ debian-release-git-remote: https://github.com/greenplum-db/debian-release.git
 debian-release-git-branch: master
 debian-package-maintainer-fullname: 'gpdb team'
 debian-package-maintainer-email: gpdb-bosh@pivotal.io
+tanzunet-client-version: v3.2.1
+tanzunet-client-git-remote: git@github.com:pivotal/gp-pivnet_client.git
+tanzunet-client-branch: master
+tanzunet-endpoint: "https://network.pivotal.io"
+tanzunet-product-slug: toolsmiths-gpdb-test
+tanzunet-metadata-file-path: ci/tanzunet/metadata.yml
+tanzunet-s3-filepath-prefix: product_files/Pivotal-Greenplum
```
Comment (xinzweb) [inline ci/concourse/vars/greenplum-database-release.prod.yml:25]: hmm, I believe we should use the `toolsmiths-gpdb-test` before we have the real slug.

**Review by xinzweb — COMMENTED** [review]
Just need to fix the variable, then it's good to go from my side.

File: ci/concourse/vars/greenplum-database-release.prod.yml (hunk 1 above)
Comment (bmdoil) [reply ci/concourse/vars/greenplum-database-release.prod.yml]: Done.

**Review by xinzweb — APPROVED** [review]
+1

**Pull request merged**